packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,2894 @@
+---
+title: Administrator Roles
+category: management
+meta:
+  - name: description
+    content: The Okta Administrator Roles API provides operations to manage administrative role assignments for a user. Read this page to get started with Admin Roles.
+---
+
+# Administrator Roles API
+
+The Okta Administrator Roles API provides operations to manage administrative role assignments for a User.
```

Comment:
This needs some sort of explanation about what it is and why it's a near copy of the existing doc.

Reply:
It also might be helpful to list and link to all the APIs that are specifically Beta.

Reply:
done